.github/autobuild/mac.sh

```diff
@@ -30,20 +30,35 @@ prepare_signing() {
     [[ -n "${MACOS_CERTIFICATE:-}" ]] || return 1
     [[ -n "${MACOS_CERTIFICATE_ID:-}" ]] || return 1
     [[ -n "${MACOS_CERTIFICATE_PWD:-}" ]] || return 1
+    [[ -n "${MACAPP_CERTIFICATE:-}" ]] || return 1
```

Comment:
This needs to be modified a bit.
I don't want the existing signing mechanism to be broken, which is what the MACOS_CERTIFICATE stuff is all about.

The App Store version needs a _different_ signing and installer signing certificates and these need to be broken out.